2023-08-XX-qubes-os-4-2-0-rc2-available-for-testing.md

```diff
@@ -0,0 +1,65 @@
+
+## What's new in Qubes 4.2.0?
+
+- Dom0 upgraded to Fedora 37
+- Xen updated to version 4.17
+- Default Debian template upgraded to Debian 12
+- Default Fedora and Debian templates use Xfce instead of GNOME
+- SELinux support in Fedora templates
+- Several GUI applications rewritten, including:
+  - Applications Menu
+  - Qubes Global Settings
+  - Create New Qube
+  - Qubes Update
+- Unified `grub.cfg` location for both UEFI and legacy boot
+- PipeWire support
+- fwupd integration for firmware updates
+- Optional automatic clipboard clearing
+- Official packages built using Qubes Builder v2
+- Split GPG and Split SSH management in Qubes Global Settings
+
+Please see the [Qubes OS 4.2.0 release notes](/doc/releases/4.2/release-notes/) for details.
+
+## When is the stable release?
+
+That depends on the number of bugs discovered in this release candidate and their severity. As explained in our [release schedule](/doc/version-scheme/#release-schedule) documentation, our usual process after issuing a new release candidate is to collect bug reports, triage the bugs, and fix them. This usually takes around five weeks, depending on the bugs discovered. If warranted, we then issue a new release candidate that includes the fixes and repeat the whole process again. We continue this iterative procedure until we're left with a release candidate that's good enough to be declared the stable release. No one can predict, at the outset, how many iterations will be required (and hence how many release candidates will be needed before a stable release), but we tend to get a clearer picture of this with each successive release candidate, which we'll share in this section in future release candidate announcements. The feedback we receive on this release candidate will determine whether another one is required.
+
+## Testing Qubes 4.2.0-rc2
+
+Thank you to everyone who tested 4.2.0-rc1! Due to your efforts, this new release candidate includes fixes for several bugs that were present in the first release candidate.
+
+If you're willing to [test](/doc/testing/) this new release candidate, you can help us improve the eventual stable release by [reporting any bugs you encounter](/doc/issue-tracking/). We encourage experienced users to join the [testing team](https://forum.qubes-os.org/t/joining-the-testing-team/5190).
+
+A full list of issues affecting Qubes 4.2.0 is available [here](https://github.com/QubesOS/qubes-issues/issues?q=is%3Aissue+label%3Aaffects-4.2). We strongly recommend [updating Qubes OS](/doc/how-to-update/) immediately after installation in order to apply all available bug fixes.
+
+## Upgrading to Qubes 4.2.0-rc2
+
+In-place upgrades from Qubes 4.1 to Qubes 4.2 are now implemented and ready for testing! As always, we strongly recommend [making a full backup](/doc/how-to-back-up-restore-and-migrate/) beforehand.
+
+Current Qubes 4.2.0-rc1 systems should be [updated normally](/doc/how-to-update/).
```

Comment:
Maybe worth mentioning updated templates here, compared to rc1?

Reply:
They're already listed [above](https://github.com/QubesOS/qubes-posts/pull/117/files#diff-09d897cff3fc9a9167181ea07660ceff91d054a93b63f5586d6b4daadb081cc7R14-R15) in the "what's new" section. I see a few options:

1. Make that section RC-specific (i.e., "What's new in Qubes 4.2.0-rc2?"). It currently just targets the stable release, even though the contents of the list do get updated with each RC, if appropriate.
2. Duplicate the content (i.e., list all changes up there and the subset that's different between RCs down here).
3. A third option that I forgot.

(2) seems messy and unnecessarily verbose to me, so I'm leaning toward (1).